testing/upload_validation.ts

```diff
@@ -192,7 +192,11 @@ export function validateSyncTableSchema(schema: any): ArraySchema<ObjectSchema<a
 function getNonUniqueElements<T>(items: T[]): T[] {
   const set = new Set<T>();
   const nonUnique: T[] = [];
-  for (const item of items) {
+  for (let item of items) {
+    // make this case insensitive for strings
+    if (typeof item === 'string') {
+      item = item.toUpperCase() as T
+    }
     if (set.has(item)) {
       nonUnique.push(item);
```

Comment:
Super nit, this will put the normalized name in the error message; the case I think we think will be most common is repeating e.g. `'Foo'` twice by accident, would be best if the original name was returned as-is. Maybe:

```typescript
for (const item of items) {
  const normalized = item.toUpperCase();
  if (set.has(normalized)) {
    nonUnique.push(item);
  }
  set.add(normalized);
}
```